frontend/src/Profile.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { user, logout, loggedoutMessage } from './reducers/user';
```

Comment:
It doesn't look like your calling your `getLoginMessage()` anywhere. I suggest you import it here...